backend/dataall/base/aws/iam.py

```diff
@@ -101,3 +100,147 @@ def delete_role_policy(
             log.error(
                 f'Failed to delete policy {policy_name} of role {role_name} : {e}'
             )
+
+    @staticmethod
+    def create_managed_policy(
+            account_id: str,
+            policy_name: str,
+            policy: str
+    ):
+        try:
+            iamcli = IAM.client(account_id)
+            response = iamcli.create_policy(
+                PolicyName=policy_name,
+                PolicyDocument=policy,
+            )
+            arn = response['Policy']['Arn']
+            log.info(
+                f'Created managed policy {arn}'
+            )
+            return arn
+        except Exception as e:
+            log.error(
+                f'Failed to create managed policy {policy_name} : {e}'
+            )
+            return None
+
+    @staticmethod
+    def delete_managed_policy_by_name(
+            account_id: str,
+            policy_name):
+        try:
+            arn = f'arn:aws:iam::{account_id}:policy/{policy_name}'
+            iamcli = IAM.client(account_id)
+            iamcli.delete_policy(
+                PolicyArn=arn
+            )
+        except Exception as e:
+            log.error(
+                f'Failed to delete managed policy {policy_name} : {e}'
+            )
+
+    @staticmethod
+    def get_managed_policy_default_version(
+            account_id: str,
+            policy_name: str):
+        try:
+            arn = f'arn:aws:iam::{account_id}:policy/{policy_name}'
+            iamcli = IAM.client(account_id)
+            response = iamcli.get_policy(PolicyArn=arn)
+            versionId = response['Policy']['DefaultVersionId']
+            policyVersion = iamcli.get_policy_version(PolicyArn=arn, VersionId=versionId)
+            policyDocument = policyVersion['PolicyVersion']['Document']
+            return versionId, policyDocument
+        except Exception as e:
+            log.error(
+                f'Failed to get policy {policy_name} : {e}'
+            )
+            return None
```

Comment:
change to `return None, None` to return tuple - otherwise may run into unpacking error

Reply:
Good catch, this leads me to think that we need to test some not-happy paths